src/packages/calendar/calendar.tsx

```
@@ -105,42 +110,64 @@ export const Calendar = React.forwardRef<
+            showTitle={showTitle}
+            showSubTitle={showSubTitle}
+            showMonthNumber={showMonthNumber}
+            scrollAnimation={scrollAnimation}
+            firstDayOfWeek={firstDayOfWeek}
+            disableDate={disableDate}
+            renderHeaderButtons={renderHeaderButtons}
+            renderDay={renderDay}
+            renderDayTop={renderDayTop}
+            renderDayBottom={renderDayBottom}
+            onConfirm={choose}
+            onDayClick={(param) => onDayClick && onDayClick(param)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)